<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ЭС Испытатель 1.1</string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/images/icon.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Испытания</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="3" rowspan="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Фильтр</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QComboBox" name="filterBox">
              <item>
               <property name="text">
                <string>Показать все</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>На испытаниях</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Выходящие сегодня</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Вышедшие с испытаний и не завершённые</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Завершённые</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3" rowspan="3">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>249</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>249</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Сводка</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item row="2" column="1">
             <widget class="QLabel" name="totalOperations">
              <property name="text">
               <string>-1</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="operationsComplete">
              <property name="text">
               <string>-1</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="currentDateTimeLabel">
              <property name="text">
               <string>currentDateTime</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="operationsDoneLabel">
              <property name="text">
               <string>Проведено: </string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="operationsLabel">
              <property name="text">
               <string>Операций:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="testerLabel">
              <property name="text">
               <string>Испытатель: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="testerName">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>noname</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QPushButton" name="settingsButton">
              <property name="text">
               <string>Настройки программы</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="operationsCompleteLabel">
              <property name="text">
               <string>Снять:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="operationsInProgress">
              <property name="text">
               <string>-1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="operationsDone">
              <property name="text">
               <string>-1</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="operationsInProgressLabel">
              <property name="text">
               <string>В процессе:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="doneTodayLabel">
              <property name="text">
               <string>Выходят сегодня:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="doneToday">
              <property name="text">
               <string>-1</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="addChipButton">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="resetSortFiltersButton">
           <property name="text">
            <string>Сбросить все фильтры и сортировки</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="importMCButton">
           <property name="text">
            <string>Импортировать МК</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="reloadChipButton">
           <property name="text">
            <string>Обновить данные</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="6" colspan="3">
          <widget class="QTableView" name="tableView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>50</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Главное меню</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="exportCSVAction"/>
    <addaction name="exportHtmlAction"/>
    <addaction name="printAction"/>
    <addaction name="separator"/>
    <addaction name="minimazeAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="testMenu">
    <property name="title">
     <string>Испытания</string>
    </property>
    <addaction name="addItemAction"/>
    <addaction name="importMCAction"/>
    <addaction name="reloadDBAction"/>
    <addaction name="resetSortFiltersAction"/>
   </widget>
   <addaction name="settingsMenu"/>
   <addaction name="testMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки программы</string>
   </property>
  </action>
  <action name="exportCSVAction">
   <property name="text">
    <string>Экспорт данных в CSV</string>
   </property>
  </action>
  <action name="minimazeAction">
   <property name="text">
    <string>Свернуть в трей</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выйти из программы</string>
   </property>
  </action>
  <action name="addItemAction">
   <property name="text">
    <string>Добавить МК</string>
   </property>
  </action>
  <action name="reloadDBAction">
   <property name="text">
    <string>Перезагрузить базу данных</string>
   </property>
  </action>
  <action name="resetSortFiltersAction">
   <property name="text">
    <string>Сбросить фильтры</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="importMCAction">
   <property name="text">
    <string>Импортировать МК</string>
   </property>
  </action>
  <action name="printAction">
   <property name="text">
    <string>Напечатать таблицу</string>
   </property>
  </action>
  <action name="exportHtmlAction">
   <property name="text">
    <string>Экспорт данных в html</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
